mobile/lib/shared/views/app_log_detail_page.dart

```diff
@@ -194,11 +127,13 @@ class AppLogDetailPage extends HookConsumerWidget {
       body: SafeArea(
         child: ListView(
           children: [
-            buildLogMessage(logMessage.message),
+            buildTextWithCopyButton("MESSAGE", logMessage.message),
+            if (logMessage.details != null)
+              buildTextWithCopyButton("DETAILS", logMessage.details.toString()),
             if (logMessage.context1 != null)
               buildLogContext1(logMessage.context1.toString()),
             if (logMessage.context2 != null)
-              buildStackMessage(logMessage.context2.toString()),
+              buildTextWithCopyButton("STACK TRACE", logMessage.context2.toString()),
```

Comment:
Can we update this to separate private widgets instead of using functions to return them?

Reply:
I have to look up what that means 😉 But I would prefer to do that in a separate PR along with various other changes I have in mind, so that this one can get merged and I can follow up with migrating other places from debugPrint to logger.

Reply:
It is not a blocking suggestion. We can do it in a separate PR, yes. Can you re-base your change over main so we can get this merged?

Reply:
> Can you re-base your change over main so we can get this merged?

Done.